service/matching/matchingEngine.go

```diff
@@ -1209,6 +1220,10 @@ func (e *matchingEngineImpl) redirectToVersionedQueueForAdd(
 		return nil, err
 	}
 	data := userData.GetData().GetVersioningData()
+	// Task queue is unversioned
+	if data == nil {
```

Comment:
Might not be the best place to put this but I couldn't get WFTs dispatched without it.

Reply:
The right fix is this: https://github.com/temporalio/temporal/blob/worker-versioning/service/matching/matchingEngine.go#L1208-L1210
If you rebase I think things should work better